src/v2/view-builder/views/IdentifierView.js

```diff
@@ -42,7 +42,20 @@ const Body = BaseForm.extend({
     if (!hasUISchemas) {
       this.$el.find('.button-primary').hide();
     }
-  }
+  },
+  showMessages () {
+    // render messages with class INFO as warning callout
+    const messagesObj = this.options.appState.get('messages');
+    if (messagesObj && messagesObj.value.length && messagesObj.value[0].class === 'INFO'
+        && this.options.appState.get('currentFormName') === 'identify') {
+      const content = messagesObj.value[0].message;
```

Comment:
ditto why only display first message?

Reply:
this was because we don't want to show all the messages if there are many.. it will look ugly. Taylor confirmed that we show just one message.

Reply:
sure. let's add comments regarding the decision.